<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="openASC">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open ASC and Start Processing</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>521</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ocean_label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ocean from -&gt; to</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ocean_level">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ocean_clip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-10000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>-3.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="ocean_label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Above ground (arma3) clip</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="aboveGround_low">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="aboveGround_high">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.400000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="aboveGround_clip">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>9999999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_from_elevation">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From Elevation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_to_elevation">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To Elevation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_change_elevation">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Elevation</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
